talamer/packages/eslint-plugin-osio/package.json

```diff
@@ -12,7 +12,7 @@
   "license": "Apache-2.0",
   "peerDependencies": {
     "eslint": "^5.7.0",
-    "prettier": "^1.14.3",
+    "prettier": "^1.15.0",
```

Comment:
why is this not `1.15.1`, typo?

Reply:
A peer typically is meant to be broad. In actuality it should probably be `*` since the package supports any prettier version.